plugins/modules/win_psrepository.py

```diff
@@ -57,6 +57,11 @@
       - I(force) has no effect when I(state=absent). See notes for additional context.
     type: bool
     default: False
+  proxy:
+    description:
+      - Proxy to use for repository.
+    type: str
+    required: no
```

Comment:
You need to add `version_added: 1.1.0` to this entry. I would do so but I don't seem to have rights to commit to your branch.

Reply:
Added `version_added: 1.1.0`